pages/integrations/sso/azure_ad.md.erb

```diff
@@ -0,0 +1,111 @@
+
+If you need to edit or update your Azure Active Directory provider settings at any time, you will need to disable the provider first. For more information on disabling a provider, see the [disabling SSO](/docs/integrations/sso#disabling-and-removing-sso) section of the SSO overview.
+
+## Using SCIM to provision and manage users
+
+<div class="Docs__note">
+  <p>This integration with AzureAD is currently available only to a limited number of customers. Contact support@buildkite.com to learn more.</p>
+</div>
+
+### Supported SCIM Features
+
+* Create Users (note that users added to Okta won't be billed for until they have signed up with Buildkite)
+* Deactivate Users (Deprovisioning)
+
+### Configuration instructions
+
+Using the SCIM provisioning settings in AzureAD, Enterprise customers can automatically remove user accounts from your Buildkite organization. In AzureAD this feature is called 'Provisioning'. You need an enabled AzureAD SSO Provider before you can set up SCIM.
+
+After creating your SSO Provider in Buildkite, you will need the  _Base URL_ and _API Token_ from your AzureAD SSO Provider Settings:
+
+<%= image "azuread-scim-settings.png", width: 1440/2, height: 548/2, alt: "Screenshot of the Buildkite Settings SCIM Deprovisioning section" %>
+
+Go to your Buildkite application in AzureAD to set up provisioning:
+
+1. Return to the Azure Admin Console and select the new 'Buildkite' enterprise app
+1. Choose 'Provisioning' from the navigation sidebar
+1. Click 'Get started'
+1. Select 'Automatic' provisioning mode and enter the following details:
+    * Tenant URL: the Base URL from your Buildkite SSO Provider settings
+    * Secret Token: the API Token from your Buildkite SSO Provider settings
+1. Click 'Test Connection'
+1. Click 'Save' once you receive confirmation the settings are valid
+1. Disable Group synchronization:
+    * Expand 'Mappings'
+    * Click 'Provision Azure Active Directory Groups'
+    * Toggle 'Enabled' to 'No' and click 'Save'
+    * Return to provisioning setttings
+1. Customise the User mappings
+    * Expand 'Mappings'
+    * Click 'Provision Azure Active Directory Users'
+    * Click the first row of the 'Attribute Mappings' table ('userPrincipalName'), change the 'Source attribute' to 'mail' and click 'OK'
+    * Delete all remaining mappings except for the 4 listed in the screenshot below:
+    * Click 'Save'
+    * Return to provisioning setttings
+1. Toggle 'Provisioning Status' to 'On'
+1. Click 'Save'
+1. Return to the `Provisioning` menu of the enterprise app and view the 'Current cycle status' section.
+    * If provisioning is working, this should say 'Initial cycle completed'
+    * If errors are displayed, click 'View provisioning logs' and for further details on what went wrong
+
+<%= image "azuread-user-mappings.png", width: 1440/2, height: 548/2, alt: "Screenshot of the required user mappings on Azure" %>
```

Comment:
The positioning of this image is a problem. It relates to a step in the middle of the above list ("Customise the User mappings").

I'm not sure we can convinced markdown to let us insert it in the middle of a list though?

Reply:
This should work, but has some super odd spacing 

```
diff --git a/pages/integrations/sso/azure_ad.md.erb b/pages/integrations/sso/azure_ad.md.erb
index a89f04a5..9cdad4d8 100644
--- a/pages/integrations/sso/azure_ad.md.erb
+++ b/pages/integrations/sso/azure_ad.md.erb
@@ -95,6 +95,7 @@ Go to your Buildkite application in AzureAD to set up provisioning:
     * Toggle 'Enabled' to 'No' and click 'Save'
     * Return to provisioning setttings
 1. Customise the User mappings
+    <%= image "azuread-user-mappings.png", width: 1440/2, height: 548/2, alt: "Screenshot of the required user mappings on Azure" %>
     * Expand 'Mappings'
     * Click 'Provision Azure Active Directory Users'
     * Click the first row of the 'Attribute Mappings' table ('userPrincipalName'), change the 'Source attribute' to 'mail' and click 'OK'
@@ -107,5 +108,4 @@ Go to your Buildkite application in AzureAD to set up provisioning:
     * If provisioning is working, this should say 'Initial cycle completed'
     * If errors are displayed, click 'View provisioning logs' and for further details on what went wrong
 
-<%= image "azuread-user-mappings.png", width: 1440/2, height: 548/2, alt: "Screenshot of the required user mappings on Azure" %>
```